app-box/app-box.html

```diff
@@ -0,0 +1,135 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../iron-flex-layout/iron-flex-layout.html">
+<link rel="import" href="../app-scroll-effects/app-scroll-effects-behavior.html">
+
+<!--
+app-box is a container that can have scroll effects.
+
+For example, the parallax effect can be used to move an image at a slower rate than the foreground.
+
+    <app-box style="height: 100px;" effects="parallax-background">
+      <img src="picture.png" style="height: 150px;" layer>
+    </app-box>
+
+@demo app-box/demo/demo1.html
+@demo app-box/demo/demo2.html
+-->
+
+<dom-module id="app-box">
+  <style>
+
+    :host {
+      display: block;
+      position: relative;
+    }
+
+    #clip {
+      @apply(--layout-fit);
+      overflow: hidden;
+      height: 100%;
+    }
+
+    #frontLayer {
+      pointer-events: none;
+      background-size: cover;
+      min-height: 100%;
+      @apply(--app-box-background);
+    }
+
+    #contentContainer {
+      position: relative;
+      height: 100%;
+      width: 100%;
+    }
+
+    :host([no-transitions]),
+    :host([no-transitions]) #frontLayer {
+      transition: none !important;
+    }
+
+  </style>
+
+  <template>
+    <div id="clip">
+      <div id="frontLayer">
+        <content select="[background]"></content>
+      </div>
+    </div>
+    <div id="contentContainer">
+      <content id="content" select=":not([background])"></content>
```

Comment:
Strictly speaking, the selector `:not([background])` is not needed, since all `[background]` elements would have been distributed at the first insertion point.